src/components/pages/ProfilePage.js

```diff
@@ -1,12 +1,54 @@
-import { React } from "react";
+import { React, useContext, useEffect, useState } from "react";
 import { useAuthenticator } from "@aws-amplify/ui-react";
 import { FaArrowRight } from "react-icons/fa";
 import Navbar from "../reusable/NavBar";
 import Sidebar from "../reusable/SideBar";
+import AlgoquantApiContext from "../../ApiContext";
+import LoadSpinner from "../reusable/LoadSpinner";
 
 const ProfilePage = () => {
   const { user } = useAuthenticator((context) => [context.user]);
   const { signOut } = useAuthenticator((context) => [context.user]);
+  const algoquantApi = useContext(AlgoquantApiContext);
+  const [jwtToken, setJwtToken] = useState(null);
+
+  const [balance, setBalance] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+  const [alpacaConnection, setAlpacaConnection] = useState(false);
+
+  // setJwtToken(user?.signInUserSession?.accessToken?.jwtToken);
+  useEffect(() => {
```

Comment:
The `JWT` should probably be set as context as well. Otherwise this code will be duplicated each time you want to use the API.

Reply:
made a jwt context / provider that is being wrapped in the pageRouter.js since that is the entry point after a user has been authenticated.